BUILD.md

```diff
@@ -39,7 +39,10 @@ You will need:
 
 * On macOS, [Xcode](https://itunes.apple.com/us/app/xcode/id497799835) with command line tools (tested with Xcode 9 and later)
 * On Linux:
-  * A C++ compiler toolchain with C++11 support (provided by `build-essential` on Ubuntu)
+  * A C++ compiler toolchain with C++11 support (`gcc` >= 4.8 or `clang` >= 3.3)
+  * `git` (typically provided by the `git` package)
+  * `patch` (typically provided by `patch` package)
```

Comment:
Just curious, do you know what this is used for?

Reply:
It's used to apply [this patch](https://github.com/apple/turicreate/blob/master/deps/patches/boost_date_time_parsers3.patch) to Boost to fix date/time parsing. There's another .patch in that directory, but I don't think that one is actually used.